2024-08-06 11:18:37 - logger_handler.py - [line:151] - ERROR - [日志信息]: 目标位置 测试数据 (T:)   ---> 存在
2024-08-06 11:18:41 - logger_handler.py - [line:151] - ERROR - [日志信息]: 网络位置: \\192.168.4.230\共享文件夹\------------项目------------\性能测试数据   ---> 存在
